Game 1 FlappyBird/FlappyBird/Source/FlappyBird/Private/Character/FlappyCharacter.cpp

```diff
@@ -0,0 +1,53 @@
+﻿// 
+
+
+#include "FlappyBird/Public/Character/FlappyCharacter.h"
+
+#include "EnhancedInputComponent.h"
+#include "EnhancedInputSubsystems.h"
+#include "Camera/CameraActor.h"
+#include "Components/CapsuleComponent.h"
+#include "GameFramework/CharacterMovementComponent.h"
+
+AFlappyCharacter::AFlappyCharacter()
+{
+	PrimaryActorTick.bCanEverTick = true;
+
+	GetCapsuleComponent()->SetSimulatePhysics(true);
+	GetCapsuleComponent()->SetEnableGravity(true);
+}
+
+void AFlappyCharacter::BeginPlay()
+{
+	Super::BeginPlay();
+	check(FlappyContext);
```

Comment:
What's this do?

Reply:
check is effectively an assert. So it will crash if FlappyContext is not set.